lib/index.js

```diff
@@ -2689,6 +2703,9 @@ function Websocket(url, opts) {
 		REACTIONS: function(channelID, messageID, reaction) {
 			return  this.MESSAGE_REACTIONS(channelID, messageID, reaction) + "/@me";
 		},
+		USER_REACTIONS: function(channelID, messageID, reaction, userID) {
```

Comment:
Change this to:

```js
USER_REACTIONS: function(channelID, messageID, reaction, userID) {
    return  this.MESSAGE_REACTIONS(channelID, messageID, reaction) + '/' + ( (!userID || userID === this.id) ? '@me' : userID );
}
```
, remove the `REACTIONS` method above, and then replace `Endpoints.REACTIONS` with `Endpoints.USER_REACTIONS` on line `390` and `418`